libbeat/cmd/instance/metrics.go

```diff
@@ -106,36 +106,81 @@ func reportBeatCPU(_ monitoring.Mode, V monitoring.Visitor) {
 	V.OnRegistryStart()
 	defer V.OnRegistryFinished()
 
-	totalCPUUsage, err := getCPUUsage()
+	totalCPUUsage, totalCPUUserTicks, totalCPUSystemTicks, totalCPUTicks, err := getCPUUsage()
```

Comment:
If we have `time`, I don't think we need ticks. Are the ticks used somewhere else?

Reply:
No, it's not used, it's just reported. I thought it could be useful as the relation CPU time and CPU ticks are different on Linux and Windows.